pkg/stub/handler.go

```diff
@@ -430,24 +430,25 @@ func (h *Handler) tbrInaccessible() bool {
 		h.imageConfigBlocksImageStreamCreation("quay.io") {
 		return true
 	}
-	err := wait.PollImmediate(5*time.Second, 20*time.Second, func() (bool, error) {
+
+	proxy := &configv1.Proxy{}
+	var err error
+	wait.PollImmediate(5*time.Second, 20*time.Second, func() (bool, error) {
```

Comment:
I think it would be worth at least logging the timeout error from this showing that it never completed.